src/node_sea.cc

```diff
@@ -33,15 +33,21 @@ using v8::Value;
 namespace node {
 namespace sea {
 
+namespace {
 // A special number that will appear at the beginning of the single executable
 // preparation blobs ready to be injected into the binary. We use this to check
 // that the data given to us are intended for building single executable
 // applications.
-static const uint32_t kMagic = 0x143da20;
+const uint32_t kMagic = 0x143da20;
 
-std::string_view FindSingleExecutableCode() {
+struct SeaResource {
+  std::string_view code;
+  bool disable_experimental_sea_warning;
```

Comment:
I think we might as well make the underlying representation a bit field now (a `uint32_t` might be good enough), as I expect that we will be adding more flags like this in the future?

Reply:
@joyeecheung done, I've also turned it into an `std::bitset` because it has a better API. PTAL.